VideoWeb/VideoWeb.Common/Caching/ConferenceCacheMapper.cs

```diff
@@ -16,10 +16,12 @@ public static Conference MapConferenceToCacheModel(ConferenceDetailsResponse con
             .Participants
             .Select(p => MapParticipantToCacheModel(p, hearingDetailsResponse))
             .ToList();
+
+        var endpointsForHearing = hearingDetailsResponse.Endpoints.ToList();
         
         var endpoints = conferenceResponse.Endpoints == null
             ? new List<Endpoint>()
-            : conferenceResponse.Endpoints.Select(EndpointCacheMapper.MapEndpointToCacheModel).ToList();
+            : conferenceResponse.Endpoints.Select(e => EndpointCacheMapper.MapEndpointToCacheModel(e, endpointsForHearing.Find(x => x.DisplayName == e.DisplayName))).ToList();
```

Comment:
it is, yeah. We need to map the interpreter language for the endpoint, but that property only exists on the bookings api model, not the video api one. So I used `endpointsForHearing.Find` to look up the bookings api version of the endpoint in order to pass it to the mapper. Hope that makes sense!

Reply:
Ah sorry, didnt see that an additional parameter has been added to the cache mapper. I think eventually we'll just map everything from booking api, or atleast when the endpoint refactor work goes in it will. Thanks for clarifying